stock_picking_backorder_strategy/README.rst

```diff
@@ -0,0 +1,128 @@
+===============================
+Picking backordering strategies
```

Comment:
@mgosai This has been changed manually. Do you plan to run the oca-gen-addon-readme tool before merging or wait for OCA bot to do it ?

Reply:
@rousseldenis We'll wait for the OCA bot to do it.